### DISCUSSION TITLE
Opentelemetry documentation for apps router showing pages router information

### DISCUSSION THREAD

**Discussion by shaunwarman** [vercel/community#5262] [Bug Report]
### Page to Investigate

https://nextjs.org/docs/app/building-your-application/optimizing/open-telemetry#getserversideprops-nextroute

### Steps to Reproduce

It looks as though the default spans referenced in the [OpenTelemetry](https://nextjs.org/docs/app/building-your-application/optimizing/open-telemetry) documentation are showing pages router information for both the pages and app router views.

On top of that, very curious how to best dig into the rendering performance of apps router. It would be great to get a trace view or a flame graph similar to the common react extension on the client side.

**Comment by amyegan** [comment]
Thanks for reporting this. I'll let the Next.js team know so we can get it fixed!
